app/controllers/state_file/questions/questions_controller.rb

```diff
@@ -33,6 +34,26 @@ def question_navigator
         end
       end
 
+      def redirect_if_no_intake
+        if current_intake.present?
+          # Assign the global id of the state_file_intake of the session to be that of the current_intake
+          # This attempts to prevent the weird timeout issues we have been experiencing in the flow.
+          # Where the session's intake does match the true current one and ends up timing out erroneously.
+          session[:state_file_intake] = "gid://vita-min/#{current_intake.class}/#{current_intake.id}"
+          # Sign out from previous intake if they differ from current_intake
+          # TODO
+        else
+          begin
+            visitor_id = cookies['visitor_id']
```

Comment:
We can't retrieve from cookies because this is written encrypted in cookies...
https://github.com/codeforamerica/vita-min/blob/main/app/controllers/concerns/state_file/start_intake_concern.rb#L13
```
    def current_intake
      @intake ||= question_navigator.intake_class.new(
        visitor_id: cookies.encrypted[:visitor_id],
        source: session[:source],
        referrer: session[:referrer]
      )
    end
```

Reply:
I nuked this code